docs/tutorials/rootless_tutorial.md

```diff
@@ -76,7 +76,9 @@ Once the Administrator has completed the setup on the machine and then the confi
 
 ### User Configuration Files.
 
-The Podman configuration files for root reside in /etc/containers.  In the rootless environment they reside in ${HOME}/.config/containers and are owned by each individual user.  The user can modify these files as they wish.
+The Podman configuration files for root reside in /etc/containers.  In the rootless environment they reside in ${XDG\_CONFIG\_HOME}/containers and are owned by each individual user.  The user can modify these files as they wish.
```

Comment:
Not strictly correct. We default our root config files to `/usr/share/containers` with overrides in `/etc/containers/`

Reply:
CHanged the first sentence to:
'The Podman configuration files for root reside in /usr/share/containers with overrides in /etc/containers. `, holler if further adjustment is necesary.